docs/content/reference/infrastructure/aws.md

```diff
@@ -0,0 +1,86 @@
+
+## Infrastructure - AWS
+
+In this section we want to dissect who creates what and what not. It contains 4 stages:
+
+- Infra pre-required and unmanaged for hypershift operator in an arbitrary AWS account
+- Infra pre-required and unmanaged in Hosted Cluster AWS account
+- Infra managed by hypershift in Management AWS account
+- Infra managed by hypershift in Hosted Cluster AWS account
+- Infra managed by kubernetes in Hosted Cluster AWS account
+
+!!! note
+    The arbitrary AWS account depends on who is providing the Hypershift service.
+
+    - **Self Managed:** It will be controlled by the [Cluster Service Provider](../concepts-and-personas.md#personas).
+    - **SaaS:** In this case the AWS Account will belong to Red Hat.
+
+#### Infra pre-required and unmanaged for hypershift operator in an arbitrary AWS account
+
+=== "Management Cluster AWS Account"
+    - 1 S3 Bucket
+        - OIDC
+        - OLM Registry for Private Hosted Cluster
```

Comment:
what is this?

Reply:
Ohh a mistake by my side, I was confusing about the bucket registry in the HC side